compiler/src/iree/compiler/InputConversion/Common/Passes.cpp

```diff
@@ -7,35 +7,62 @@
 #include "iree/compiler/InputConversion/Common/Passes.h"
 
 #include "iree/compiler/Dialect/Flow/Transforms/Passes.h"
+#include "iree/compiler/Dialect/Util/Transforms/Passes.h"
 #include "mlir/Pass/Pass.h"
 #include "mlir/Pass/PassManager.h"
 #include "mlir/Pass/PassOptions.h"
 #include "mlir/Pass/PassRegistry.h"
 
-namespace mlir::iree_compiler {
+namespace mlir::iree_compiler::InputConversion {
 
 namespace {
 #define GEN_PASS_REGISTRATION
 #include "iree/compiler/InputConversion/Common/Passes.h.inc" // IWYU pragma: export
 } // namespace
 
-void buildCommonInputConversionPassPipeline(OpPassManager &passManager) {
+void buildCommonInputConversionPassPipeline(
+    OpPassManager &passManager, const TransformOptions &transformOptions) {
   passManager.addPass(createIREEImportPublicPass());
   passManager.addPass(createImportMLProgramPass());
   passManager.addPass(createSanitizeModuleNamesPass());
+
+  // TODO: this pass should either live in InputConversion or be run in flow -
+  // it's a mistake that it's here.
   passManager.addPass(IREE::Flow::createConvertMeshToFlowPass());
```

Comment:
Outdated TODO? (This is in InputConversion)

Reply:
New TODO - the pass lives in flow but is added in input conversion - it should either live in InputConversion or be run in flow. I have no clue why it was done like this.

Reply:
Ah, gotcha.